custom/bha/commcare_extensions.py

```diff
@@ -0,0 +1,17 @@
+from django.conf import settings
+
+from corehq.apps.case_search.fixtures import custom_csql_fixture_context
+from corehq.messaging.templating import SimpleDictTemplateParam
+
+from .util import get_user_clinic_ids, get_user_facility_ids
+
+BHA_DOMAINS = settings.CUSTOM_DOMAINS_BY_MODULE['custom.bha']
+
+
+@custom_csql_fixture_context.extend(domains=BHA_DOMAINS)
+def bha_csql_fixture_context(domain, restore_user):
+    facility_ids = get_user_facility_ids(domain, restore_user)
+    return ('bha', SimpleDictTemplateParam({
+        'facility_ids': facility_ids,
```

Comment:
Does this mean the facility_ids are evaluated every time even when they are not used in the CSQL?

Reply:
Yep, though only for BHA.  The fixture context is generated once per restore though, so if any indicators in the fixture use it, it's not wasted effort.  If we think this param (or future ones) aren't actually needed all the time, we could probably write a custom template param that lazily fetches expensive fields on demand.